.github/workflows/yaml-lint.yml

```
@@ -1,6 +1,9 @@
----
 name: Yaml Lint
-on: [push]
+on:
+  pull_request:
+    paths:
+      - '.github/**.yaml'
+      - '.github/**.yml'
```

```suggestion
      - '**/*.yaml'
      - '**/*.yml'
```

I guess maybe that would be better? We also need lint `.licenserc.yaml` etc.

---

Can you verify this?

---

ok